examples/multilang-multi-project/proto/lib/google/protobuf/map_proto2_unittest.proto

```
@@ -0,0 +1,98 @@
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+syntax = "proto2";
+
+// We don't put this in a package within proto2 because we need to make sure
+// that the generated code doesn't depend on being in the proto2 namespace.
+// In map_test_util.h we do "using namespace unittest = proto2_unittest".
+package proto2_unittest;
+
+import "google/protobuf/unittest_import.proto";
```

Buf (1.54.0)

import "google/protobuf/unittest_import.proto": file does not exist

(COMPILE)